code/z_adventurezones/lavamoon.dm

```diff
@@ -1110,29 +1116,29 @@ var/global/iomoon_blowout_state = 0 //0: Hasn't occurred, 1: Moon is irradiated
 
 				return 0
 
-
-	rotor
-		name = "giant rotors"
+	base
+		name = "huge contraption"
 		desc = "An enormous artifact of some sort. You feel uncomfortable just being near it. Probably because it is a giant piece of dangerous machinery."
+		anchored = ANCHORED
+		density = 0
 		icon = 'icons/effects/160x160.dmi'
-		icon_state = "powercore_rotors_off"
+		icon_state = "powercore_base"
 		bound_height = 160
 		bound_width = 160
-		layer = 3.9 // TODO layer
-		density = 0
+		layer = 3.2 // TODO layer
 
 		var/obj/iomoon_boss/core/core = null
 
-	base
-		name = "huge contraption"
-		desc = "An enormous artifact of some sort. You feel uncomfortable just being near it."
+	top
+		name = "mechanism core"
+		desc = "An enormous artifact of some sort. You feel uncomfortable just being near it. Probably because it is a giant piece of dangerous machinery."
 		anchored = ANCHORED
 		density = 0
 		icon = 'icons/effects/160x160.dmi'
-		icon_state = "powercore_base"
+		icon_state = "powercore_core"
 		bound_height = 160
 		bound_width = 160
-		layer = 3.7 // TODO layer
+		layer = 7 // TODO layer
 
```

Comment:
```suggestion
plane = PLANE_NOSHADOW_BELOW
```

I'm not totally sure if BELOW or ABOVE is appropriate but this stops the inappropriate depth shadow.

The same line should be added to the core as well.